### PR TITLE
YaruBanner: adapt inner to outer border radius

### LINKED ISSUE #186: YaruBanner: adapt inner to outer border radius
It is something not related to this PR, but the inkwell border radius looks a bit weird.

_Originally posted by @Jupi007 in https://github.com/ubuntu/yaru_widgets.dart/pull/185#pullrequestreview-1084902893_

### PR DESCRIPTION
**Pull request by Jupi007**
image

@Feichtmeier Do you think we can export the extension? I think this can be useful in many contexts.

Fixes #186

### CODE REVIEW

**Review by Feichtmeier — APPROVED** [review]
Very nice solution! (Dart is so cool xD)

I think exporting is a nice idea! This situation occurs quiet often

**Comment by Jupi007** [comment]
Okay, I'll do it this evening. I'm currently not at home.

And yes, dart is super cool 😎 
This extension feature is incredible.

File: lib/src/utilities/yaru_banner.dart [hunk 1]
```diff
@@ -62,7 +63,7 @@ class YaruBanner extends StatelessWidget {
     return InkWell(
       onTap: onTap,
       borderRadius: borderRadius,
-      hoverColor: Theme.of(context).colorScheme.onSurface.withOpacity(0.1),
```
Comment (Feichtmeier) [inline lib/src/utilities/yaru_banner.dart:65]: woops xD

Comment (Jupi007) [reply lib/src/utilities/yaru_banner.dart:65]: How did I do that 😅 sorry.

Comment (Feichtmeier) [reply lib/src/utilities/yaru_banner.dart:65]: np I fixed it :paintbrush:

Comment (Jupi007) [reply lib/src/utilities/yaru_banner.dart:65]: Thanks :)

**Pull request merged**